ipapython/certdb.py

```diff
@@ -180,12 +178,11 @@ def unparse_trust_flags(trust_flags):
 
 
 def verify_kdc_cert_validity(kdc_cert, ca_certs, realm):
-    pem_kdc_cert = kdc_cert.public_bytes(serialization.Encoding.PEM)
-    pem_ca_certs = '\n'.join(
-        cert.public_bytes(serialization.Encoding.PEM) for cert in ca_certs)
+    pem_ca_certs = ''.join(
```

Comment:
This should have been bytes, I will make this uses the `x509.write_certificate_list` function instead.